.travis.yml

```diff
@@ -54,8 +54,8 @@ matrix:
       env: LIBRABBITMQ_VERSION=v0.5.2 TEST_PHP_ARGS=-m
 
 before_install:
-  - sh provision/install_rabbitmq-c.sh ${LIBRABBITMQ_VERSION}
-  - if [ "${COVERITY_SCAN_BRANCH}" != 1 ]; then phpize && ./configure  --with-librabbitmq-dir=$HOME/rabbitmq-c && make ; fi
+  - sudo ./provision/install_rabbitmq-c.sh
+  - if [ "${COVERITY_SCAN_BRANCH}" != 1 ]; then phpize && ./configure && make ; fi
```

Comment:
I removed the `--with-librabbitmq-dir` option cause the rabbitmq-c library is now installed globally.

Reply:
I guess `${LIBRABBITMQ_VERSION}` was removed by accident. Or there was any other changes to build system I missed?

Reply:
`${LIBRABBITMQ_VERSION}` is used to access to an environment variable.

Before: this environment variable was passed as an argument in the installation script
Now: The script directly use the environment variable

As said, if you prefer I can update the script to use an argument instead of the environment variable directly.

Reply:
The idea to install rabbitmq-c locally is to avoid using `sudo` in CI so that we could utilize container-based travis build env, [which is faster](https://blog.travis-ci.com/2014-12-17-faster-builds-with-container-based-infrastructure/). I guess we already have `sudo: false` in `.travis-ci.yml` so unless there is hard requirement to install globally, I would avoid sudo and go with local install.

Reply:
@odolbeau oh, I'm sorry, missed that. I'd suggest to pass version explicitly. The rationale for that is that this script is copied time to time by others for their CI and for internal  purposes, so I'd prefer to make their life easier.

If you decide to bring argument passing back, could you also add this check to install script

```
if [ "$#" -ne 1 ]; then
  echo "Usage: $0 LIBRABBITMQ_VERSION" >&2
  exit 1
fi
```